### PR TITLE
added tamer to advisors

### PR DESCRIPTION
**Pull request by a-wong-8**
Screenshot 2024-11-25 at 9 57 56 AM

### CODE REVIEW

**Comment by amilkh** [comment]
looks good for now! 
maybe a bit long text tho

File: src/intl/en/team.json
Comment (gevuong) [inline src/intl/en/team.json:1]: Does this file need to be updated? If so, why?

1. It appears to work the same locally.
2. the other `team.json` files in the other language folders (ie. `/es`, `/pt`, `/ja`) were not updated.

**Comment by a-wong-8** [comment]
Screenshot 2024-11-25 at 3 09 23 PM

File: src/intl/en/team.json
Comment (a-wong-8) [reply src/intl/en/team.json:1]: I couldnt remember which one to change so I changed them both

**Pull request merged**